jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/loginform/JmixLoginI18n.java

```diff
@@ -50,6 +52,14 @@ public static JmixLoginI18n createDefault() {
         return JsonSerializer.toObject(JmixLoginI18n.class, JMIX_DEFAULT_I18N);
     }
 
+    public void setForm(JmixForm form) {
```

Comment:
if we call `com.vaadin.flow.component.login.LoginI18n#setForm`, then `getForm()` will return `null`. Override super `setForm` and check that passed parameter is instance of `JmixForm`, otherwise throw illegal argument exception

Reply:
We can create new JmixForm instance and init it by LoginI18n.getForm. In this case rememberMe field will be empty. What is correct.

If we override setForm, then we'll get that IllegalState exception. Because the argument of the method has type LoginI18n.Form, and exactly this type will be created when converting Json to Object.